packages/clerk-js/src/core/auth/safeLock.ts

```diff
@@ -10,6 +10,20 @@ export function SafeLock(key: string) {
   });
 
   const acquireLockAndRun = async (cb: () => unknown) => {
+    if ('locks' in navigator && isSecureContext) {
+      const controller = new AbortController();
+      const lockTimeout = setTimeout(() => controller.abort(), 4999);
```

Comment:
Curious about the `4999` here. 🤔

Reply:
It was one ms less than the the 5s polling interval 😅